medusa/providers/torrent/json/torrentday.py

```diff
@@ -96,9 +97,9 @@ def search(self, search_strings, age=0, ep_obj=None, **kwargs):
                     continue
 
                 try:
-                    jdata = response.json()
-                except ValueError:
-                    log.debug('No data returned from provider')
+                    jdata = json.loads(response.content.decode('unicode-escape'))
```

Comment:
Think about `{"name": "Escaped chars test: \"torrent\". It\'s dangerous..."}`.
```python
>>> print ur'{"name": "Escaped chars test: \"torrent\". It\'s dangerous..."}'
{"name": "Escaped chars test: \"torrent\". It\'s dangerous..."}
>>> print ur'{"name": "Escaped chars test: \"torrent\". It\'s dangerous..."}'.decode('unicode-escape')
{"name": "Escaped chars test: "torrent". It's dangerous..."}
```

Maybe just replace all `r"\'"` (= `"\\'"`)  with `"'"`?

Reply:
In my opinion that's error-prone.  A batteries included option: `pip install dirtyjson`
```python
import dirtyjson as json
json.loads(response.content)
```

Reply:
@sharkykh You're likely not doing any `dump(s)` in provider code, so importing as json shouldn't be a problem, however per the zen of python "Explicit is better than implicit." so keeping it as dirtyjson is reasonable.

Reply:
Done. Thanks for the good suggestions 👍